src/makeCalculator.js

```diff
@@ -4,7 +4,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add(value) {
+      result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      result -= value;
+
+      return this;
+    },
+
+    multiply(value) {
+      result *= value;
+
+      return this;
+    },
+    divide(value) {
+      result /= value;
+
+      return this;
+    },
+    operate(callback, value) {
+      return callback.call(this, value);
```

Comment:
[CODE STYLE] - The `operate` method should use the arithmetic operators with the assignment operator `=` to modify the `result` value. Additionally, the method should return `this` to allow chaining.